electronicparsers/wien2k/parser.py

```diff
@@ -1011,6 +1050,11 @@ def parse_method(self):
             sec_method.x_wien2k_ifft = fft[:3]
             sec_method.x_wien2k_ifft_factor = fft[3]
 
+        # find inc file and set as a mainfile for Wien2kParser
+        inc_file = self.get_wien2k_file('inc')
+        self.inc_parser.mainfile = inc_file
+        # TODO: handle when inc_file is None
```

Comment:
Is the TODO still relevant? I see that the whole corehole parsing is inside `if self.inc_parser.mainfile:`.

Reply:
Right, thanks for pointing that out.

Reply:
Corrected.